meta-resin-common/recipes-support/resin-hostname/resin-hostname/resin-hostname

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -e
+
+. /usr/sbin/resin-vars
+
+if [ -z "$NEW_HOSTNAME" ]; then
+    NEW_HOSTNAME=$(jq -r '.uuid //empty' $CONFIG_PATH)
```

Comment:
Why is this check performed here? If you do this here then every script sourcing `resin-vars` will have the wrong `NEW_HOSTNAME`